### PR TITLE
Clarify that egress gateway is disabled by default if Helm was used to install

### LINKED ISSUE #3398: Egress Example Failing With Helm Install
Step 5 in https://preliminary.istio.io/docs/examples/advanced-gateways/egress-gateway-tls-origination/ is failing if you use the helm install option due to the fact that values.yaml sets enabled to false. If you reinstall with the egress gateway enabled (helm template $HOME/istio-fetch/istio --name istio --namespace istio-system --set gateways.istio-egressgateway.enabled=true | kubectl apply -f -), then this step passes successfully. 

Also, the docs concerning validating the installation also indicate that the egress gateway should be present, but in the case of a helm install, the deployment is never created.

### PR DESCRIPTION
**Pull request by brian-avery**
Fixes #3398

### CODE REVIEW

File: content/docs/examples/advanced-gateways/egress-gateway-tls-origination/index.md [hunk 1]
```diff
@@ -16,6 +16,9 @@ TLS origination for traffic to external services.
 
 *   Setup Istio by following the instructions in the [Installation guide](/docs/setup/).
 
+    {{ <tip> }}
+    If you used Helm to install, then the egress gateway is disabled by default. Please enable it and reinstall Istio before continuing.
```
Comment (frankbu) [inline content/docs/examples/advanced-gateways/egress-gateway-tls-origination/index.md]: Probably should be more clear: `If you used Helm with default values to install Istio`

File: content/docs/setup/kubernetes/quick-start/index.md [hunk 2]
```diff
@@ -96,11 +96,15 @@ Follow our instructions on how to
...
 
 1.  To ensure the following Kubernetes services are deployed: `istio-citadel`,
-    `istio-engressgateway`, `istio-galley`, `istio-ingress`, `istio-ingressgateway`,
+    `istio-egressgateway`, `istio-galley`, `istio-ingress`, `istio-ingressgateway`,
     `istio-pilot`, `istio-policy`, `istio-statsd-prom-bridge`, `istio-telemetry`,
     `prometheus`, and optionally, `istio-sidecar-injector`, verify they all have
     an appropriate `CLUSTER-IP`:
 
+    {{< tip >}}
+    Note that istio-egress gateway is not present by default if you have used Helm to install.
```
Comment (frankbu) [inline content/docs/setup/kubernetes/quick-start/index.md]:
```
Note that `istio-egressgateway` is not present
```

File: content/docs/examples/advanced-gateways/egress-gateway-tls-origination/index.md [hunk 3]
```diff
@@ -16,6 +16,9 @@ TLS origination for traffic to external services.
...
+    {{ </tip> }}
```
Comment (frankbu) [inline content/docs/examples/advanced-gateways/egress-gateway-tls-origination/index.md]: add blank line after this

File: content/docs/examples/advanced-gateways/egress-gateway-tls-origination/index.md (hunk 1 above)
Comment (brian-avery) [reply content/docs/examples/advanced-gateways/egress-gateway-tls-origination/index.md]: Fixed. Thanks.

File: content/docs/setup/kubernetes/quick-start/index.md (hunk 2 above)
Comment (brian-avery) [reply content/docs/setup/kubernetes/quick-start/index.md]: Fixed. Thanks.

File: content/docs/examples/advanced-gateways/egress-gateway-tls-origination/index.md (hunk 3 above)
Comment (brian-avery) [reply content/docs/examples/advanced-gateways/egress-gateway-tls-origination/index.md]: Fixed. Thanks.

File: content/docs/examples/advanced-gateways/egress-gateway-tls-origination/index.md [hunk 4]
```diff
@@ -16,6 +16,10 @@ TLS origination for traffic to external services.
 
 *   Setup Istio by following the instructions in the [Installation guide](/docs/setup/).
 
+    {{< tip >}}
+    If you used Helm with default values to install Istio, then the egress gateway is disabled by default. Please enable it and reinstall Istio before continuing.
```
Comment (rcaballeromx) [inline content/docs/examples/advanced-gateways/egress-gateway-tls-origination/index.md]: Where can a reader find the instructions to perform those tasks?

File: content/docs/setup/kubernetes/quick-start/index.md [hunk 5]
```diff
@@ -94,9 +94,12 @@ Follow our instructions on how to
 [use Helm and Tiller to manage the Istio deployment](/docs/setup/kubernetes/helm-install/#option-2-install-with-helm-and-tiller-via-helm-install).
 
 ## Verifying the installation
+{{< tip >}}
+Note that istio-egressgateway is not present by default if you used Helm to install.
```
Comment (rcaballeromx) [inline content/docs/setup/kubernetes/quick-start/index.md]: Do not add "note" to a callout, it's redundant. The text doesn't provide enough context. Tips work best if you place the reason for the tip at the beginning. 
```suggestion
If you installed Istio using Helm, the `istio-egressgateway` service is not present by default. Consider this fact as you verify that the services are deployed.
```

File: content/docs/examples/advanced-gateways/egress-gateway-tls-origination/index.md (hunk 4 above)
Comment (brian-avery) [reply content/docs/examples/advanced-gateways/egress-gateway-tls-origination/index.md]: Fixed. Thanks.

**Comment by geeknoid** [comment]
Deploy preview for *preliminary-istio* processing.

Building with commit bae78588d336bf59463407ffa675a3db1a798782

https://app.netlify.com/sites/preliminary-istio/deploys/5c76a49d2bfcfa0008c168f7

**Comment by brian-avery** [comment]
Any update on this PR?

**Comment by frankbu** [comment]
I think this PR can be closed, since the issue is being addressed by other PRs.
^^^ @vadimeisenbergibm

**Comment by vadimeisenbergibm** [comment]
@frankbu @brian-avery The part in 
https://github.com/istio/istio.io/pull/3399/files#diff-3a94fb91fcc15390adb11f3e2e8e382d should be removed, the other two files are relevant.

**Comment by brian-avery** [comment]
There's nothing in regards to verification or anything in the helm document that I see so I don't think the section stating that the egressgateway isn't present is needed there anymore. Since the 3534 was merged, I think the only thing that we need now is to update the default values.

**Pull request merged**